.npmignore

```diff
@@ -1,11 +0,0 @@
-*.log
-npm-debug.log*
-coverage
-.nyc_output
-*.tgz
-.env
-*.swp
-*.vim
-.npm
-.tap-output
-Jenkinsfile
```

Comment:
What's reason to remove the npmignore file?

Reply:
instead of a denylist for files that should _not_ be included in the package, the `files` key in `package.json` specifies files to include explicitly.

Reply:
oh, I missed that, makes sense.